src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -71,20 +113,87 @@ public void stop() {
     spinWheels(0);
   }
 
+  public void hold() {
+    spinWheels(isConeMode ? kConeIntakeSpeed : kCubeIntakeSpeed);
+  }
+  
+  public Command pullTimed(double seconds, boolean isConeMode){
+    return new StartEndCommand(this::pull, this::stop, this).withTimeout(seconds).beforeStarting(isConeMode ? this::setConeMode : this::setCubeMode);
+  }
+
+  public Command pushTimed(double seconds, boolean isConeMode){
```

Comment:
Use a different parameter name for isConeMode so it isn't confused with the class variable.